buildozer/__init__.py

```diff
@@ -935,8 +935,7 @@ def usage(self):
 
             if not meth.__doc__:
                 continue
-            doc = [x for x in
-                    meth.__doc__.strip().splitlines()][0].strip()
+            doc = list(meth.__doc__.strip().splitlines())[0].strip()
```

Comment:
It's not too clear to me why list comprehension are not recommended here :thinking: 
Could you share the analyser recommendations about it?

Reply:
there is no transformation, the list comprehension is effectively used as a way to cast into a list, so i agree it's cleaner to just do that, the code is both shorter and more idiomatic, and i have the feeling it could be faster, or at least, could be made faster by python, than the list comprehension, which is a loop by construction, while casting doesn't have such constraint.

Reply:
@AndreMiras here is detailed recommendations by analyzer:

> It is unnecessary to use a comprehension just to loop over the iterable and create a list/set/dict out of it. Python has a specialized set of tools for this task: the list/set/dict constructors, which are faster and more readable.
> Not Preferred:

states = [
    ('AL', 'Alabama'),
    ('AK', 'Alaska'),
    ('AZ', 'Arizona'),
    ('AR', 'Arkansas'),
    ('CA', 'California'),
    # ...
]

abbreviations_to_names = {
    abbreviation: name
    for abbreviation, name in states
}
Preferred:

states = [
    ('AL', 'Alabama'),
    ('AK', 'Alaska'),
    ('AZ', 'Arizona'),
    ('AR', 'Arkansas'),
    ('CA', 'California'),
    # ...
]

abbreviations_to_names = dict(states)

Reply:
Thank you both, completely makes sense :+1: